File: src/Processors/Twitter/Integrations/Analysis.py
class Analysis():
    def __init__(self, polarity, subjectivity, subject, messages, date):
        self.polarity = polarity
        self.subjectivity = subjectivity
        self.subject = subject
        self.messages = messages
        self.date = date

    def toDict(self):
        return {
            'polarity': self.polarity,
            'subjectivity': self.subjectivity,
            'subject': self.subject,
            'messages': self.messages,
            'date': self.date
        }

File: src/WebServices/Twitter/Entities/Analysis.cs
namespace Twitter.Entities
{

    using System;
    using MongoDB.Bson;
    using MongoDB.Bson.Serialization.Attributes;

    public class Analysis
    {
        [BsonId]
        public ObjectId _id { get; set; }

        public string date { get; set; }

        public long messages { get; set; }

        public double subjectivity { get; set; }

        public double polarity { get; set; }

        public string subject { get; set; }
    }
}